packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -1022,8 +1054,14 @@ export class DeltaManager
         connection.on("error", this.errorHandler);
         connection.on("pong", this.pongHandler);
 
-        const initialMessages = connection.initialMessages;
+        const initialMessages = connection.initialMessages.sort((a, b) => a.sequenceNumber - b.sequenceNumber);
```

Comment:
Is this `initialMessages` property [this code](https://github.com/microsoft/FluidFramework/blob/cd3c8a8f3419f0dd3f68fba4f73024dfad4a12e8/packages/drivers/driver-base/src/documentDeltaConnection.ts#L187). It looks like it should already be sorted if so?

Reply:
It is a combination of "early" ops and initialMessages from PUSH. Because of that it could be unsorted - I think I was able to hit this case